Makefile

```diff
@@ -9,7 +9,8 @@ update_all:
 	git submodule update --recursive --remote
 
 REL_PATH_OF_THIS_MAKEFILE:=$(lastword $(MAKEFILE_LIST))
-ABS_ROOT_DIR:=$(abspath $(dir $(REL_PATH_OF_THIS_MAKEFILE)))
+# use cygpath -m because Coq on Windows cannot handle cygwin paths
+ABS_ROOT_DIR:=$(shell cygpath -m '$(ABS_ROOT_DIR)' 2>/dev/null || echo '$(ABS_ROOT_DIR)')
```

Comment:
This looks like a cyclic definition, is it what you meant? Does it actually become an absolute path, or is it just taking the empty string, which happens to denote the current directory, relatively?

Reply:
I believe it is cyclic if you use delayed assignment (`=`), and sensible if you use eager assignment (`:=`).  I may have swapped which assignment symbol is which kind of assignment, and perhaps this deserves a comment?  Or we could just not mutate the value and use a different variable name the second time.

Reply:
Oh, oops, I did not mean to delete the line above this one

Reply:
Oh, oops, I did not mean to delete the line above this one